### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by kpal81xd**
- Sets default develop script to use debug engine (skips terser also)
- Proper importing of PCUI
- Updated playcanvas 
- Removed unused modules
- Reordered tsconfig
- Refactored custom plugin
- Removed caching and use unbundled engine version for treeshaking

### CODE REVIEW

**Comment by slimbuck** [comment]
Looks like you've removed the ability to specify custom PCUI package for development, i.e.:
`PCUI_PATH=../pcui npm run develop`.

File: src/viewer.ts [hunk 1]
```diff
@@ -489,9 +501,12 @@ class Viewer {
...
             const p = node.getPosition();
-            if (p.x < min_x) min_x = p.x; else if (p.x > max_x) max_x = p.x;
-            if (p.y < min_y) min_y = p.y; else if (p.y > max_y) max_y = p.y;
-            if (p.z < min_z) min_z = p.z; else if (p.z > max_z) max_z = p.z;
+            if (p.x < min_x) min_x = p.x;
+            else if (p.x > max_x) max_x = p.x;
+            if (p.y < min_y) min_y = p.y;
+            else if (p.y > max_y) max_y = p.y;
+            if (p.z < min_z) min_z = p.z;
+            else if (p.z > max_z) max_z = p.z;
```
Comment (kungfooman) [inline src/viewer.ts:509]: This was readable before :sweat_smile:

Comment (willeastcott) [reply src/viewer.ts:509]: Presumably this is Prettier doing its thing...

**Review by willeastcott — APPROVED** [review]

**Comment by slimbuck** [comment]
Isn't pcui remapping still broken since you're importing directly from `@playcanvas/pcui/react`?

**Pull request merged**